### PR TITLE
Us156404 implement discard button in ai box

### PR DESCRIPTION
**Pull request by JamesVaChao**
Implment discard, localized, sparkle icon

Rally Story
[US156404](https://rally1.rallydev.com/#/?detail=/userstory/719624512167&fdp=true): Implement Discard button in AI box

### CODE REVIEW

File: lang/en.js [hunk 1]
```diff
@@ -437,19 +437,32 @@ export default {
...
 	"practice:llm:editSource:line1": "The source text is copied from your page content.",
 	"practice:llm:editSource:line2": "Editing this text will not alter your page content.",
 	"practice:llm:editSource:line3": "If edited, you can reset this text to the original page content.",
 	"practice:llm:editSource:line4": "Source text must be in English between {min}-{max} characters.",
 	"practice:llm:editSource:label": "Edit Source Text",
+
+	"practice:llm:generated:error": "An error has occurred",
+	"practice:llm:generated:sourceTextInfo" : "This question is based on the source text provided",
+
+	"practice:llm:generating:sourceTextInfo": "Generating X question based on source text:",
```
Comment (mdgbayly) [inline lang/en.js:454]: nit: I think you left in my extraneous `X`. I think I added that at one point when trying to work out why my BSI wasn't dynamically reloading my changes on browser refresh...

Comment (JamesVaChao) [reply lang/en.js:454]: removed, thought it was as placeholder

**Review by mdgbayly — APPROVED** [review]
LGTM

**Comment by grant-cleary** [comment]
Oops, didn't notice this was a merge into another branch, sorry. Ignore my approval; I'll take a peek once the main branch is ready to merge.

**Review by mdgbayly — APPROVED** [review]

**Pull request merged**